Source/Core/DolphinQt2/Debugger/BreakpointWidget.cpp

```diff
@@ -40,8 +40,15 @@ BreakpointWidget::BreakpointWidget(QWidget* parent) : QDockWidget(parent)
     if (!Settings::Instance().IsDebugModeEnabled())
       return;
 
+    m_new->setEnabled(Core::IsRunning());
     m_load->setEnabled(Core::IsRunning());
     m_save->setEnabled(Core::IsRunning());
+    if (!Core::IsRunning())
```

Comment:
Any thoughts on using `state` here instead of calling Core::IsRunning() 4 times?

Reply:
Or at least cache the result in a local?